core-services/prow/02_config/_config.yaml

```diff
@@ -2263,6 +2263,7 @@ tide:
     - openshift/aws-account-shredder
     - openshift/aws-efs-operator
     - openshift/azure-misc
+    - openshift/boilerplate
```

Comment:
how did this sneak in?

Reply:
oops, must have been a bad rebase (its a commit from master)